Tests/Tests.Sync.Shared/SyncTestHelpers.cs

```diff
@@ -57,7 +57,10 @@ public static Task<User> GetUserAsync()
             return new SyncConfiguration(user, serverUri);
         }
 
-        public static Task<User> GetFakeUserAsync(string token = "foo:bar", string scheme = "http") => User.LoginAsync(Credentials.AdminToken(token), new Uri($"{scheme}://some.fake.server:9080"));
+        public static Task<User> GetFakeUserAsync(string token = "foo:bar", string scheme = "http")
```

Comment:
We should still pass an identity. Let's add another optional parameter `string identity = null` and in the body `identity = identity ? Guid.NewGuid().ToString();`.